Sources/Contentful/Decodable.swift

```diff
@@ -144,30 +144,43 @@ internal class LinkResolver {
 
     // Caches the callback to resolve the relationship represented by a Link at a later time.
     internal func resolve(_ link: Link, inLocale localeCode: LocaleCode, callback: @escaping (Any) -> Void) {
-        callbacks[DataCache.cacheKey(for: link, with: localeCode)] = callback
+        let key = DataCache.cacheKey(for: link, with: localeCode)
+        if callbacks[key] == nil {
+            callbacks[key] = [callback]
+        } else {
+            callbacks[key]!.append(callback)
+        }
```

Comment:
Can we do this in a more safe way without force unwrapping?
```
if let callbacksForKey = callbacks[key] {
    callbacksForKey.append(callback)
} else {
    callbacks[key] = [callback]
}
```

Reply:
This won't work. The new local variable is not a reference to the original, but a copy of the value. You do realize that this isn't an unsafe force unwrap, correct? This is common practice immediately after checking that something is not nil, especially if the type is an Array or Dict. Anyway, I've changed them to `?` to make them "more safe" even though that's less efficient.

Reply:
I overlooked that the callbacks array was being copied, but you're totally correct.

Thanks again for the contribution. I'll add some tests before merging. A `1.0.0-beta1` should be arriving sometime in the coming weeks and we can finally have some API stability around here.